ffmpeg-plugin/clone-and-patch-ffmpeg.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright 2024 Intel Corporation
+
+set -e
+
+if [ -n "$1" ]; then
+	ffmpeg_ver=$1
+else
+	# default to latest 6.1
+	ffmpeg_ver=6.1
+fi
+
+rm FFmpeg -rf
+git clone https://github.com/FFmpeg/FFmpeg.git
+cd FFmpeg
+git checkout release/"$ffmpeg_ver"
+git apply ../"$ffmpeg_ver"/*.patch
+cd ..
```

Comment:
```suggestion
rm "${SCRIPT_DIR}/FFmpeg" -rf
git clone https://github.com/FFmpeg/FFmpeg.git "${SCRIPT_DIR}/FFmpeg"
git -C "${SCRIPT_DIR}/FFmpeg" checkout release/"$ffmpeg_ver"
git -C "${SCRIPT_DIR}/FFmpeg" apply "${SCRIPT_DIR}/FFmpeg/${ffmpeg_ver}/*.patch"
```

Reply:
Let's improve the script in a separate PR. I suggest not to amend this one.